### PR TITLE
Update solutionsLib.py

### PR DESCRIPTION
**Pull request by bikesbade**
#110

### CODE REVIEW

File: scripts/solutionsLib.py [hunk 1]
```diff
@@ -1064,11 +1064,11 @@ def executeCommand(self, com, index=0):
                     self.getProcessInfoValue(processKey, 'max_cached_scale', index))
                 if os.path.isfile(tileSchemeMtc):
                     cachepath = os.path.join(self.getProcessInfoValue(processKey, 'in_cache_location', index), self.getProcessInfoValue(processKey, 'in_cache_name', index))
-                    lodNodesList = returnLevelDetails(os.path.join(cachepath, 'conf.xml'))
+                    lodNodesList = self.returnLevelDetails(os.path.join(cachepath, 'conf.xml'))
                     lodNodesList = sorted(lodNodesList, key=lambda k: int(k['level']))
                     maxLODNode = lodNodesList[-1]
                     maxScale = maxLODNode['scale']
-                    modifyConfProperties(os.path.join(cachepath, 'conf.properties'), maxScale)
+                    self.modifyConfProperties(os.path.join(cachepath, 'conf.properties'), maxScale)
```
Comment (Chamlika) [inline scripts/solutionsLib.py:1071]: @bikesbade  Put the whole block in a try:  exception block to trap any errors in (returnLevelDetails, modifyConfProperties) functions. I see the XMLs loaded & modified without any error checks within the functions. Please return False if an exception is thrown with an appropriate error message logged to signal w/f failures. Thanks!

**Review by Chamlika — CHANGES_REQUESTED** [review]
Please read the comment, thanks! @bikesbade

**Comment by bikesbade** [comment]
@Chamlika made the changes as requested. Thanks

**Review by vijaygit02 — APPROVED** [review]

**Comment by chritomix** [comment]
Hi, with @bikesbade. I just have downloaded the latest script and it is not working. With your last update of module solutionsLib.py, you made a mistake with indentation. you must move "def modifyConfProperties(self, properties, maxScale)" function to the left. OTHERWISE THE MDSC is failing. thanks, Tomasz

**Comment by bikesbade** [comment]
@chritomix  Thanks for pointing it out. Made the fixed. you can test it out. Thanks

**Pull request merged**